ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -209,6 +209,7 @@ export default function BuildQuote ({
           selectPlaceHolderText="Select"
           hideItemIf={(item) => item.address === selectedToToken?.address}
           listContainerClassName="build-quote__open-dropdown"
+          autoFocus="true"
```

Comment:
Since this is now a `bool`, I think the `"true"` is no longer needed here:
```suggestion
          autoFocus
```

Reply:
Good shout, left over from testing :(